src/routes/authorization.js

```diff
@@ -0,0 +1,19 @@
+const Router = require('koa-router');
+const jwt = require('jsonwebtoken');
+
+const secret = process.env.SECRET;
+
+export const router = new Router({ prefix: '/auth' });
+
+router
+  .post('/signin', async (ctx, next) => {
+    const { header, payload } = ctx.request.body;
+
+    if (payload.login === '36545fktrcfylh' && payload.password === '36545fktrcfylh') {
+      ctx.body = { token: jwt.sign({ header, payload }, secret) };
```

Comment:
Why header is here?

Reply:
It is

{
   "alg": "HS256"
}

Where it should be?

Reply:
🤦‍♂ What is first argument in `sign` method?